terraform/plugin.go

```diff
@@ -30,6 +30,7 @@ func Plugin(ctx context.Context) *plugin.Plugin {
 			"terraform_output":      tableTerraformOutput(ctx),
 			"terraform_provider":    tableTerraformProvider(ctx),
 			"terraform_resource":    tableTerraformResource(ctx),
+			"terraform_module":      tableTerraformModule(ctx),
```

Comment:
Could you please sort the tables like https://github.com/turbot/steampipe-plugin-pagerduty/blob/main/pagerduty/plugin.go#L23-L36